tests/baselines/reference/outModuleConcatES6.js

```diff
@@ -12,6 +12,11 @@ export class B extends A { }
 
 //// [all.js]
 // This should be an error
+export class A {

```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.